### DISCUSSION TITLE
Solucion QMP3

### DISCUSSION THREAD

**Discussion by felipenani** [dds-jv/foro#370] [Consultas generales] [answered]
Buenos días, con el grupo queríamos comenzar con la cuarta iteración del QMP pero antes corregir lo que teníamos hecho en la tercera iteración para no arrastrar posibles errores. Sin embargo, no encontramos en ninguna parte una posible solución de la misma.
Desde ya, muchas gracias y saludos.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @felipenani 

Te paso un boceto de la solución 3: https://docs.google.com/document/d/1eMm7sRyyi-PJ8YFt0Ro2ecxfHBC6WJBlui7x21rFA2Q/edit
